package.json

```diff
@@ -18,12 +18,12 @@
     "type-check": "tsc --noEmit"
   },
   "keywords": [
-    "cypress",
-    "plugin"
+    "cucumber-js",
+    "cypress"
   ],
   "repository": {
     "type": "git",
-    "url": "git://github.com/onthespotqa/cypress-plugin-macros.git"
+    "url": "https://github.com/onthespotqa/cypress-plugin-macros"
   },
```

Comment:
"files": ['lib'] needs to be added to the package so the npm publish only publishes that directory.